### PR TITLE
Add documentation for frigidaire AC

### PR DESCRIPTION
**Pull request by bm1549**
## Proposed change

Add docs for a new integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

 
- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/51136
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2608
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/frigidaire.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Frigidaire
+description: Instructions on how to integrate Frigidaire smart AC within Home Assistant.
+ha_category:
+  - Presence Climate
+ha_release: 2021.06
+ha_iot_class: Cloud Polling
+ha_quality_scale: silver
```
Comment (bm1549) [inline source/_integrations/frigidaire.markdown:8]: This is speculative, let me know if this is not the right thing to do and I can remove this line

File: source/_examples/configuration_yaml_by_silvrrgit.markdown [hunk 2]
```diff
@@ -1,6 +0,0 @@
----
-title: "Configuration.yaml by SilvrrGIT"
-description: ""
-ha_category: Example configuration.yaml
```
Comment (klaasnicolaas) [inline source/_examples/configuration_yaml_by_silvrrgit.markdown]: Why is this change in your PR?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by klaasnicolaas** [comment]
Please rebase you base branch to the `next` branch to solve the unrelated commit in your PR.

File: source/_examples/configuration_yaml_by_silvrrgit.markdown (hunk 2 above)
Comment (bm1549) [reply source/_examples/configuration_yaml_by_silvrrgit.markdown]: I had my local branch based on the default (`current`) but opened up the PR against `next`. I just rebased and force pushed up a change to fix this

File: source/_integrations/frigidaire.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/frigidaire.markdown:8]: This data is copied from the integration manifest in the upstream codebase. It is reviewed there (and automatically imported into the documentation on release).

File: source/_integrations/frigidaire.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ha_domain: frigidaire
+---
+
+[Frigidaire](https://www.frigidaire.com/) is a company that creates air conditioners with an app to control them.
+
+The Frigidaire integration allows you to turn on/off, change mode, change fan speed, and change temperature of the air conditioners.
+
+## Prerequisites
+
+To use the integration you must own at least one Frigidaire Smart AC and have credentials to login to the app.
```
Comment (frenck) [inline source/_integrations/frigidaire.markdown]: I think we can remove this as a Prerequisite...

File: source/_integrations/frigidaire.markdown [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ha_release: 2021.06
+ha_iot_class: Cloud Polling
+ha_quality_scale: silver
+ha_config_flow: true
+ha_codeowners:
+  - "@bm1549"
+ha_domain: frigidaire
+---
+
+[Frigidaire](https://www.frigidaire.com/) is a company that creates air conditioners with an app to control them.
```
Comment (frenck) [inline source/_integrations/frigidaire.markdown]:
```suggestion
[Frigidaire](https://www.frigidaire.com/) creates air conditioners with an app to control them.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/frigidaire.markdown (hunk 3 above)
Comment (bm1549) [reply source/_integrations/frigidaire.markdown]: Removed

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**